### ISSUE TITLE
Add PlaygroundEquipmentList - Materialized View

### ISSUE THREAD

**Issue by Tapir79** [ORGTYTESPOO/kiradigiasm#34]
Playground equipments as list grouped by park name, address and model.

**Comment by ernogispo** [comment]
I will make a pull request with this grouping. However, in a KAKE-digi blog post (https://kakedigi.wordpress.com/2017/06/26/sql-relaatiotietokannan-suorituskyvyn-optimointia/) also brand is mentioned.

**Comment by Tapir79** [comment]
Correct, brand is an attribute that is needed as well. Thanks for pointing this out. I'll add this.
